de.metas.business.rest-api/src/main/java/de/metas/rest_api/invoicecandidates/response/JsonCloseInvoiceCandidatesResponseItem.java

```diff
@@ -0,0 +1,41 @@
+package de.metas.rest_api.invoicecandidates.response;
+
+import de.metas.rest_api.common.MetasfreshId;
+import de.metas.util.lang.ExternalId;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+/*
+ * #%L
+ * de.metas.business.rest-api
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+@Value
+@Builder
+public class JsonCloseInvoiceCandidatesResponseItem
+{
+	ExternalId externalHeaderId;
```

Comment:
Are externalHeaderId and externalLineId displayed nicely in swagger if they are declared like this?
Note that i switched to JsonExternalId in the createInvoiceCandidates-EP. Maybe that's alternative here, too

Reply:
You are right. It's nicer with the new object because we don't see that " value"  in the response.
Changed it for Close and also for the Status